Rectangle.java

```diff
@@ -0,0 +1,38 @@
+public class Rectangle {
+    private int hight;
+    private int width;
+
+    public Rectangle(){
+
+    }
+
+    public Rectangle(int y, int x){
+        this.hight=y;
+        this.width=x;
+    }
+
+    public void findPerimeter(){
+        System.out.println("Периметр прямоугольника == "+(2*(hight+width)));

```

Comment:
Pls use space between math sign and variable
' " + (2 * (height + width)) ' is reading easier than without space.